test/models/tenant_test.rb

```diff
@@ -13,30 +15,29 @@ class TenantTest < ModelCase
 
   test '#usage_count returns the expected amount when the tenant has already been used' do
     tenant = create :tenant
-    Redis.current.set tenant.id, 10
+    tenant_calls = 10
+
+    tenant_calls.times.each { |_n| tenant.register_usage }
 
-    assert 10 == tenant.usage_count
+    assert tenant.usage_count == tenant_calls
   end
 
   test '#usage_count returns the expected amount when the tenant has never been used' do
     tenant = create :tenant
-    Redis.current.set tenant.id, nil
 
-    assert 0 == tenant.usage_count
+    assert tenant.usage_count.zero?
   end
 
-  test '#register_usage increments the tenant usage by 1'  do
+  test '#register_usage increments the tenant usage by 1' do
     tenant = create :tenant
-    Redis.current.set tenant.id, 0
```

Comment:
This was actually a bug, I believe - because I didn't see this happening outside of your tests so the API wouldn't actually do this when you register a tenant on it's own.

Reply:
Nice catch!  It'd only work without initializing because Redis allows `INCR` operation on `null` values to act as if you were incrementing a `0`. But relying on the DB provider is definitely dangerous. 

![image](https://user-images.githubusercontent.com/9383701/140835793-e9ecedc0-db8b-4961-b21e-cf3180ffd40e.png)

Reply:
Very nice, I just didn't like that we were setting it to 0 in tests.  I hadn't realized that.